src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/pyMCsingleSegStime_NoLoop.f90

```diff
@@ -6,17 +6,14 @@ module muskingcunge_interface
 implicit none
 contains
 subroutine c_muskingcungenwm(dt, qup, quc, qdp, ql, dx, bw, tw, twcc,&
-    n, ncc, cs, s0, velp, depthp, qdc, velc, depthc) bind(c)
+    n, ncc, cs, s0, velp, depthp, qdc, velc, depthc, ck, cn, X) bind(c)
 
     real(c_float), intent(in) :: dt
     real(c_float), intent(in) :: qup, quc, qdp, ql
     real(c_float), intent(in) :: dx, bw, tw, twcc, n, ncc, cs, s0
     real(c_float), intent(in) :: velp, depthp
     real(c_float), intent(out) :: qdc, velc, depthc
-    real(c_float) :: ck, cn, X
-    !TODO: Incorporate ck, cn, X into v02 output;
-    ! these are currently dropped silently from
-    ! the interface output.
+    real(c_float), intent(out) :: ck, cn, X
```

Comment:
Here, the silent dropping of `ck`, `cn`, and `X` is reversed. These variables are now declared as `intent(out)` and included in the `c_muskingcungenwm` argument list. **This is the upstream most change**